### PR TITLE
Feature accurate hour display

### LINKED ISSUE #46: Fix error message in reports form
If you type over 24 huors in reports form, error message shows "(...) 24.00".
Should be "(...) 24:00".

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Resolves #46

Related #106

### CODE REVIEW

File: employees/serializers.py [hunk 1]
```diff
@@ -50,3 +50,10 @@ class Meta:
             'description',
             'work_hours',
         )
+
+    def to_representation(self, instance):
+        data = super(ReportSerializer, self).to_representation(instance)
```
Comment (kbeker) [inline employees/serializers.py]: As far as I know in python3 you don't have to write `super(Class, self)`. `super()` should be enough

File: employees/serializers.py [hunk 2]
```diff
@@ -50,3 +50,10 @@ class Meta:
...
+        # SELF NOTE: For some reason, tests for list get methods got a Queryset as instance in serializer and I dunno why...
+        if type(self.instance) is Report:
```
Comment (kbeker) [inline employees/serializers.py]: change it to 
```python
if isinstance(self.instance, Report):
```

File: employees/serializers.py [hunk 3]
```diff
@@ -50,3 +50,10 @@ class Meta:
             'description',
             'work_hours',
         )
+
+    def to_representation(self, instance):
+        data = super(ReportSerializer, self).to_representation(instance)
+        # SELF NOTE: For some reason, tests for list get methods got a Queryset as instance in serializer and I dunno why...
```
Comment (kbeker) [inline employees/serializers.py]: Add it as a issue and in have to be investigate later. After adding issue please delete comment.

Comment (maciejSamerdak) [reply employees/serializers.py]: image

File: employees/serializers.py (hunk 1 above)
Comment (maciejSamerdak) [reply employees/serializers.py]: I've seen it around the web. Thought that maybe it's some sort of good practise, like explicit parametrising in function calls.

Comment (maciejSamerdak) [reply employees/serializers.py]: Done

File: employees/serializers.py (hunk 2 above)
Comment (maciejSamerdak) [reply employees/serializers.py]: Done

File: employees/serializers.py (hunk 3 above)
Comment (maciejSamerdak) [reply employees/serializers.py]: Done

**Pull request merged**